airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/spec.json

```diff
@@ -34,12 +76,71 @@
       }
     }
   },
-  "authSpecification": {
-    "auth_type": "oauth2.0",
-    "oauth2Specification": {
-      "rootObject": [],
-      "oauthFlowInitParameters": [],
-      "oauthFlowOutputParameters": [["access_token"]]
+  "advanced_auth": {
```

Comment:
why are all of these new fields needed in the spec?

Reply:
this fields are used for auth with `origin` field